examples/ephemeral-spark/elasticsearch.tf

```
@@ -29,9 +29,10 @@ module "sg-ports-es" {
 }
 
 module "aws-sg-es" {
-  source              = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
-  vpc_id              = var.vpc_id
-  ingress_cidr_blocks = var.ingress_cidr_blocks
+  source                  = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  vpc_id                  = var.vpc_id
+  ingress_cidr_blocks     = var.ingress_cidr_blocks
+  ingress_security_groups = concat(module.aws-sg-vm.security_group_ids, module.aws-emr-sg-master.security_group_ids, module.aws-emr-sg-core.security_group_ids)
```

We'll want spark's security group instead of HBase
```suggestion
  ingress_security_groups = concat(module.aws-sg-vm.security_group_ids, [module.ephemeral-spark-sgs.emr_managed_sg_id])
```